root/defaults/default

```diff
@@ -1,7 +1,7 @@
 server {
-	listen 80;
-	location / {
-		root /assets;
-		autoindex on;
-	}
+    listen ${NGINX_PORT};
+    location / {
+        root /assets;
+        autoindex on;
+    }
```

Comment:
Indent adjustments and port for server made configurable using templating system (done before start-up)